packages/familie-backend/src/router.ts

```diff
@@ -32,6 +32,7 @@ export default (
     router.get(
         '/user/profile',
         ensureAuthenticated(true, saksbehandlerTokenConfig),
+        hentBrukerenhet(saksbehandlerTokenConfig),
```

Comment:
I dont think we should fetch enhet each time. It can be done when the user logs in and then stored in the session.

Reply:
yes, so the hentBrukerenhet() will first check if the enhet has been stored in session and only fetch if not found